### PR TITLE
Add in a config option to turn off auto-shutdown.

### PR DESCRIPTION
**Pull request by knappe**

### CODE REVIEW

**Comment by dmvt** [comment]
:shipit: also with a README update

**Comment by knappe** [comment]
@ryanwi @dmvt Same question about the format of the Changelog.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Emque Producing CHANGELOG
```
Comment (ryanwi) [inline CHANGELOG.md]: you mean consuming here?

**Comment by ryanwi** [comment]
👍 as well with a readme update on usage

**Pull request merged**